docs-gen/content/rule_set/basics.md

```diff
@@ -103,22 +103,37 @@ Vehicle.Navigation.CurrentLocation:
 
 It is recommended for servers, which are implementing protocols for the vehicle signal specification, to serve old and new nodes during the deprecation period described above.
 
+## Naming Rules
+
+Different languages and environments have different rules for identifiers.
+To simplify usage of VSS in as many languages and environments as possible the following rules apply.
+
+* The allowed character set for node names (branches and signals) is `A-Z`, `a-z` and `0-9`
+* Node names must not start with a digit
+* Names are case sensitive, but it is not allowed to have two nodes that only differ by case,
+  i.e. it is not allowed to have both `Vehicle.Aaa` and `Vehicle.AaA`
+* It is not allowed to reuse the same node name as a parent node in a child node,
```

Comment:
This reads like if you mean the immediate parent, as opposed to all parents.

Reply:
Will try to rephrase it